modules/adding-bare-metal-host-to-cluster-using-yaml.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/managing-bare-metal-hosts.adoc
+
+[id="adding-bare-metal-host-to-cluster-using-yaml_{context}"]
+= Adding a bare metal host to the cluster using YAML in the web console
+
+You can add bare metal hosts to the cluster in the web console using a YAML file that describes the bare metal host.
+
+.Prerequisites
+
+* Install a {op-system} compute machine on bare metal infrastructure for use in the cluster.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. In the web console, navigate to *Compute* -> *Bare Metal Hosts*.
+. Select *Add Host* -> *New with YAML*.
```

Comment:
It should be `New from YAML` 
<img width="181" alt="Screen Shot 2021-06-16 at 10 18 51 AM" src="https://user-images.githubusercontent.com/12692381/122147862-4ac37980-ce8c-11eb-8bea-74adb4ec426a.png">